### PR TITLE
Global environment

### PR DESCRIPTION
**Pull request by rebekaburnett**

### CODE REVIEW

**Comment by rebekaburnett** [comment]
I haven't tried to use it yet - will try tomorrow. I need to get the other PRs through the system first.

File: tools/vipps-api-global-postman-environment.json [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+			"enabled": true
+		},
+		{
+			"key": "amount",
+			"value": "499",
+			"enabled": true
+		},
+		{
+			"key": "amount_capture",
+			"value": "0",
```
Comment (cloveras) [inline tools/vipps-api-global-postman-environment.json:32]: Yes!

File: tools/vipps-api-global-postman-environment.json [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+			"enabled": true
+		},
+		{
+			"key": "shippingMethod2",
+			"value": "Postman Method 2",
+			"enabled": true
+		},
+		{
+			"key": "transactionTextInitiate",
+			"value": "Transaction initiated through Postman",
```
Comment (cloveras) [inline tools/vipps-api-global-postman-environment.json:107]:
```suggestion
			"value": "This payment was initiated with Postman.",
```

File: tools/vipps-api-global-postman-environment.json [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+			"enabled": true
+		},
+		{
+			"key": "transactionTextCapture",
+			"value": "Transaction captured through Postman",
```
Comment (cloveras) [inline tools/vipps-api-global-postman-environment.json:112]:
```suggestion
			"value": "This payment was captured with Postman.",
```

File: tools/vipps-api-global-postman-environment.json [hunk 4]
```diff
@@ -0,0 +1,364 @@
...
+			"enabled": true
+		},
+		{
+			"key": "transactionTextRefund",
+			"value": "Transaction refunded through Postman",
```
Comment (cloveras) [inline tools/vipps-api-global-postman-environment.json:117]:
```suggestion
			"value": "This payment was refunded with Postman.",
```

File: tools/vipps-api-global-postman-environment.json [hunk 5]
```diff
@@ -0,0 +1,364 @@
...
+			"enabled": true
+		},
+		{
+			"key": "transactionTextCancel",
+			"value": "Transaction cancelled through Postman",
```
Comment (cloveras) [inline tools/vipps-api-global-postman-environment.json:122]:
```suggestion
			"value": "This payment was cancelled with Postman.",
```

File: tools/vipps-api-global-postman-environment.json [hunk 6]
```diff
@@ -0,0 +1,364 @@
...
+		{
+			"key": "well-known_uri",
+			"value": "\"Add url to well-known endpoint for used environment\"",
+			"description": "Url to well-known endpoint for th OIDC service",
+			"enabled": true
+		},
+		{
+			"key": "redirect_uri",
+			"value": "\"Add redirect uri\"",
+			"description": "Redirect uri to merchant",
```
Comment (cloveras) [inline tools/vipps-api-global-postman-environment.json:139]:
```suggestion
			"description": "Redirect URI to the merchant.",
```

File: tools/vipps-api-global-postman-environment.json [hunk 7]
```diff
@@ -0,0 +1,364 @@
...
+			"enabled": true
+		},
+		{
+			"key": "merchantRedirectUrl",
+			"value": "https://example.com/",
+			"enabled": true
+		},
+		{
+			"key": "merchantAgreementUrl",
+			"value": "https://example.com/",
```
Comment (cloveras) [inline tools/vipps-api-global-postman-environment.json:222]:
```suggestion
			"value": "https://example.com/agreement-url",
```

File: tools/vipps-api-global-postman-environment.json (hunk 5 above)
Comment (rebekaburnett) [reply tools/vipps-api-global-postman-environment.json:122]: Nice! @cloveras  just commit as you go :-)

File: tools/vipps-api-global-postman-environment.json [hunk 8]
```diff
@@ -0,0 +1,364 @@
...
+			"enabled": true
+		},
+		{
+			"key": "price",
+			"value": "1000",
```
Comment (cloveras) [inline tools/vipps-api-global-postman-environment.json:227]:
```suggestion
			"value": "49900",
```

File: tools/vipps-api-global-postman-environment.json [hunk 9]
```diff
@@ -0,0 +1,364 @@
...
+			"enabled": true
+		},
+		{
+			"key": "due",
+			"value": "2040-01-01",
+			"enabled": true
+		},
+		{
+			"key": "description_initial",
+			"value": "Transaction initiated through Postman",
```
Comment (cloveras) [inline tools/vipps-api-global-postman-environment.json:303]:
```suggestion
			"value": "This payment was initiated with Postman.",
```

File: tools/vipps-api-global-postman-environment.json [hunk 10]
```diff
@@ -0,0 +1,364 @@
...
+			"enabled": true
+		},
+		{
+			"key": "description_charge",
+			"value": "Charged through Postman",
```
Comment (cloveras) [inline tools/vipps-api-global-postman-environment.json:308]:
```suggestion
			"value": "This payment was charged with Postman.",
```

File: tools/vipps-api-global-postman-environment.json [hunk 11]
```diff
@@ -0,0 +1,364 @@
...
+			"enabled": true
+		},
+		{
+			"key": "productDescription_updated",
+			"value": "Updated product created through Postman",
```
Comment (cloveras) [inline tools/vipps-api-global-postman-environment.json:313]:
```suggestion
			"value": "This product was updated with Postman",
```

File: tools/vipps-api-global-postman-environment.json [hunk 12]
```diff
@@ -0,0 +1,364 @@
...
+			"enabled": true
+		},
+		{
+			"key": "productDescription",
+			"value": "Product created through Postman",
```
Comment (cloveras) [inline tools/vipps-api-global-postman-environment.json:318]:
```suggestion
			"value": "This payment was created with Postman.n",
```

File: tools/vipps-api-global-postman-environment.json [hunk 13]
```diff
@@ -0,0 +1,364 @@
...
+			"enabled": true
+		},
+		{
+			"key": "description_refund",
+			"value": "Charge refunded through Postman",
+			"enabled": true
+		},
+		{
+			"key": "productName_updated",
+			"value": "Updated postman test product",
```
Comment (cloveras) [inline tools/vipps-api-global-postman-environment.json:328]:
```suggestion
			"value": "Updated Postman test product",
```

File: tools/vipps-api-global-postman-environment.json [hunk 14]
```diff
@@ -0,0 +1,364 @@
+{
+	"id": "c4c3427b-fd37-4431-b4fb-a175cfba74ff",
+	"name": "Vipps API Global Environment v1",
```
Comment (cloveras) [inline tools/vipps-api-global-postman-environment.json:3]: Nice.

File: tools/vipps-api-global-postman-environment.json [hunk 15]
```diff
@@ -0,0 +1,364 @@
...
+			"enabled": true
+		},
+		{
+			"key": "fallBack",
+			"value": "https://example.com/",
+			"enabled": true
+		},
+		{
+			"key": "shippingDetailsPrefix",
+			"value": "https://example.com/",
```
Comment (cloveras) [inline tools/vipps-api-global-postman-environment.json:52]:
```suggestion
			"value": "https://example.com/vipps/shipping",
```

File: tools/vipps-api-global-postman-environment.json [hunk 16]
```diff
@@ -0,0 +1,364 @@
...
+			"enabled": true
+		},
+		{
+			"key": "consentRemovalPrefix",
+			"value": "https://example.com/",
```
Comment (cloveras) [inline tools/vipps-api-global-postman-environment.json:57]:
```suggestion
			"value": "https://example.com/vipps/consent-removal",
```

File: tools/vipps-api-global-postman-environment.json [hunk 17]
```diff
@@ -0,0 +1,364 @@
...
+		},
+		{
+			"key": "orderId",
+			"value": "<This value will be overwritten on each Initiate Payment call with MSN + timestamp + random int>",
+			"enabled": true
+		},
+		{
+			"key": "vippsLandingPageURL",
+			"value": "<This value will be overwritten on each Initiate Payment call. It will contain a url to the Vipps landing page, obtained from the eCom api.>",
+			"enabled": true
```
Comment (cloveras) [inline tools/vipps-api-global-postman-environment.json:83]: Does `enabled` mean that it's send in requests, or something else? We can skip sending this as it's an optional parameter. If we omit it, it probably causes less confusion than sending it.

**Comment by rebekaburnett** [comment]
@cloveras Thank you!!

**Pull request merged**